future-timeout/Cargo.toml

```diff
@@ -0,0 +1,7 @@
+[package]
+name = "future-timeout"
```

Comment:
Smallest crate in the universe :D

Perhaps just duplicate this?

Reply:
Hahaha fair enough.